includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -202,6 +227,35 @@ public function prepare_items() {
 		);
 	}
 
+	/**
+	 * Generate values needed for users table headers totals.
+	 * Values: $total_courses_started, $total_active_courses, $total_average_grade.
+	 *
+	 * @since  4.2.0
+	 */
+	private function set_users_table_header_total_values() {
+
+		// Get average grade for all the lessons graded for all the courses.
+		$this->total_average_grade = Sensei_Grading::get_graded_lessons_average_grade();
```

Comment:
I'm not sure it's worthwhile setting this here since it's just a one-liner. We might as well get rid of the variable and call this function directly in `get_columns`.